### DISCUSSION TITLE
Translate to another language than English

### DISCUSSION THREAD

**Discussion by hassanayoub1985** [openai/whisper#566] [Q&A] [answered]
Hello,
This command translates audios to English:
whisper French.wav --language French --task translate

How to translate to another language than English like French (So, the output of English.wav will be in French text)?

Thanks

**Comment by jongwook** [comment, marked as answer]
The model is not trained for that; it occasionally does it when you set `task="translate"` and `language` as the target language, but it's mostly due to imperfect language id in the training data, and it won't be very reliable.
